server/fileupdater.mjs

```diff
@@ -339,3 +340,25 @@ function v9NumericStringSort(routine) {
     if(typeof routine[key] === 'string')
       routine[key] = routine[key].replace('numericSort', 'numericStringSort');
 }
+
+function v10UpdateCountParameter(routine) {
+  for(const key in routine) {
+    if(routine[key] && [ 'FLIP', 'MOVE', 'MOVEXY', 'ROTATE' ].indexOf(routine[key].func) != -1 && typeof routine[key].count != 'undefined') {
+      if(!routine[key].count) {
+        routine[key].count = 'all';
+      } else if(typeof routine[key].count == 'string') {
```

Comment:
I guess the purpose of this branch is to convert `''` to `'all'`? Otherwise, I'm not sure what the purpose of this branch is.

Reply:
Actually the purpose is to remove the default that 0 converts to 999999.

At the moment you cannot move 0 cards

This causes a problem that sometimes with automations you get a variable that gets value 0 and all of a sudden you move all cards.

So now you can either you "all" or just use a big number to move all cards but you can also "move" 0 cards

Reply:
sorry I misunderstood the question. This if is to check if someone used a "${ }" which would be a major problem. in this case it generates an IF routine that simulates the previous behavior